extensions/wikia/FlowTracking/scripts/createNewPageFlowsVE.js

```diff
@@ -0,0 +1,9 @@
+require(['wikia.flowTracking.createPage', 'mw', 'jquery'], function (flowTrackingCreatePage, mw, $) {
+	function init() {
+		mw.hook('ve.activationComplete').add(function () {
+			flowTrackingCreatePage.trackOnEditPageLoad('ve');
```

Comment:
As we are using 'sourceedit', 'rte-source' and 'rte-wysiwyg' I'm wondering if we should use 've' or maybe 'visualeditor'. What do you think?